Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -9610,7 +9595,7 @@
     "Context": {
       "ReferenceDateTime": "2018-10-24T12:00:00"
     },
-    "Comment": "FORMAT not properly handled. Works with spaces around middle -",	
+    "Comment": "FORMAT not properly handled. Works with spaces around middle -",
```

Comment:
Is the the only left case from the ones with the tags mentioned in the issue?

Reply:
ah, I did not notice, I missed some other  "FORMAT" case, I will fix and push them here.

Reply:
@LionbridgeCSII, do you have an estimate when do you think you can push the fixes to this PR? Thanks.

Reply:
@tellarin, the fixes are done, but there is still an issue in the Java RegExpUtility which gives problems because it wrongly handles negative lookbehinds.
I am trying to fix it, but if you prefer, I can push the current fixes and open a new issue for this problem.

Reply:
Please push the current changes, so we can take a look. We can decide then how to proceed.

Reply:
@tellarin, these changes have also been pushed.